nucypher/cli/commands/felix.py

```diff
@@ -165,14 +198,10 @@ def felix():
 @option_discovery_port(default=FelixConfiguration.DEFAULT_LEARNER_PORT)
 @group_config_options
 def init(general_config, config_options, config_root, discovery_port):
-    """
-    Create a brand-new Felix.
-    """
-    emitter = _setup_emitter(general_config, config_options.checksum_address)
-
+    """Create a brand-new Felix."""
+    emitter = setup_emitter(general_config=general_config, banner=config_options.checksum_address)
```

Comment:
```suggestion
    emitter = setup_emitter(general_config=general_config, banner=FELIX_BANNER.format(config_options.checksum_address or ''))
```

Same for other Felix commands as well.